glommio/src/task/raw.rs

```diff
@@ -204,36 +204,27 @@ where
 
```

Comment:
Just curious on this commit message... is `Jiang` a tool of some kind like Valgrind? Or a human being reporting an issue?

Reply:
Jiang is one of our contributors, that contributed our mesh implementation